ccl/importccl: TestImportData failed under stress

SHA: https://github.com/cockroachdb/cockroach/commits/f38e4c7a2bb3f0e697b25d21bdd8ea67b163c32f

Parameters:
```
TAGS=
GOFLAGS=
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=787510&tab=buildLog

```
=== RUN   TestImportData
W180720 05:22:08.529272 69 server/status/runtime.go:143  Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I180720 05:22:08.535046 69 server/server.go:795  [n?] monitoring forward clock jumps based on server.clock.forward_jump_check_enabled
I180720 05:22:08.537912 69 server/config.go:552  [n?] 1 storage engine initialized
I180720 05:22:08.537950 69 server/config.go:555  [n?] RocksDB cache size: 128 MiB
I180720 05:22:08.537964 69 server/config.go:555  [n?] store 0: in-memory, size 0 B
I180720 05:22:08.552740 69 server/node.go:421  [n?] **** cluster 79815005-6dab-4be9-bf0d-e3ba2d786de4 has been created
I180720 05:22:08.552775 69 server/server.go:1370  [n?] **** add additional nodes by specifying --join=127.0.0.1:39727
I180720 05:22:08.553495 69 storage/store.go:1480  [n1,s1] [n1,s1]: failed initial metrics computation: [n1,s1]: system config not yet available
I180720 05:22:08.553566 69 server/node.go:553  [n1] initialized store [n1,s1]: disk (capacity=512 MiB, available=512 MiB, used=0 B, logicalBytes=6.9 KiB), ranges=1, leases=0, writes=0.00, bytesPerReplica={p10=7043.00 p25=7043.00 p50=7043.00 p75=7043.00 p90=7043.00 pMax=7043.00}, writesPerReplica={p10=0.00 p25=0.00 p50=0.00 p75=0.00 p90=0.00 pMax=0.00}
I180720 05:22:08.553604 69 server/node.go:399  [n1] node ID 1 initialized
I180720 05:22:08.553726 69 gossip/gossip.go:363  [n1] NodeDescriptor set to node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:39727" > attrs:<> locality:<> ServerVersion:<major_val:2 minor_val:0 patch:0 unstable:9 > 
I180720 05:22:08.553908 69 storage/stores.go:222  [n1] read 0 node addresses from persistent storage
I180720 05:22:08.554011 69 server/node.go:703  [n1] connecting to gossip network to verify cluster ID...
I180720 05:22:08.554486 69 server/node.go:728  [n1] node connected via gossip and verified as part of cluster "79815005-6dab-4be9-bf0d-e3ba2d786de4"
I180720 05:22:08.554525 69 server/node.go:487  [n1] node=1: started with [<no-attributes>=<in-mem>] engine(s) and attributes []
I180720 05:22:08.554711 69 server/config.go:336  [n1] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180720 05:22:08.554731 69 server/server.go:1799  [n1] Could not start heap profiler worker due to: directory to store profiles could not be determined
I180720 05:22:08.554778 69 server/server.go:1507  [n1] starting https server at 127.0.0.1:36357
I180720 05:22:08.554794 69 server/server.go:1508  [n1] starting grpc/postgres server at 127.0.0.1:39727
I180720 05:22:08.554806 69 server/server.go:1509  [n1] advertising CockroachDB node at 127.0.0.1:39727
W180720 05:22:08.554822 69 sql/jobs/registry.go:299  [n1] unable to get node liveness: node not in the liveness table
I180720 05:22:08.566226 329 storage/replica_command.go:279  [split,n1,s1,r1/1:/M{in-ax}] initiating a split of this range at key /System/"" [r2]
I180720 05:22:08.574779 335 storage/replica_command.go:279  [split,n1,s1,r2/1:/{System/-Max}] initiating a split of this range at key /System/NodeLiveness [r3]
I180720 05:22:08.580169 273 storage/replica_command.go:279  [split,n1,s1,r3/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/NodeLivenessMax [r4]
I180720 05:22:08.585701 318 storage/replica_command.go:279  [split,n1,s1,r4/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/tsd [r5]
I180720 05:22:08.590605 362 storage/replica_command.go:279  [split,n1,s1,r5/1:/{System/tsd-Max}] initiating a split of this range at key /System/"tse" [r6]
I180720 05:22:08.595050 373 storage/replica_command.go:279  [split,n1,s1,r6/1:/{System/tse-Max}] initiating a split of this range at key /Table/SystemConfigSpan/Start [r7]
I180720 05:22:08.602381 384 storage/replica_command.go:279  [split,n1,s1,r7/1:/{Table/System…-Max}] initiating a split of this range at key /Table/11 [r8]
I180720 05:22:08.608346 395 storage/replica_command.go:279  [split,n1,s1,r8/1:/{Table/11-Max}] initiating a split of this range at key /Table/12 [r9]
I180720 05:22:08.618822 418 storage/replica_command.go:279  [split,n1,s1,r9/1:/{Table/12-Max}] initiating a split of this range at key /Table/13 [r10]
I180720 05:22:08.621871 404 sql/event_log.go:126  [n1,intExec=optInToDiagnosticsStatReporting] Event: "set_cluster_setting", target: 0, info: {SettingName:diagnostics.reporting.enabled Value:true User:root}
I180720 05:22:08.633869 450 storage/replica_command.go:279  [split,n1,s1,r10/1:/{Table/13-Max}] initiating a split of this range at key /Table/14 [r11]
I180720 05:22:08.638943 119 sql/event_log.go:126  [n1,intExec=set-setting] Event: "set_cluster_setting", target: 0, info: {SettingName:version Value:$1 User:root}
I180720 05:22:08.640789 279 storage/replica_command.go:279  [split,n1,s1,r11/1:/{Table/14-Max}] initiating a split of this range at key /Table/15 [r12]
I180720 05:22:08.645671 464 sql/event_log.go:126  [n1,intExec=disableNetTrace] Event: "set_cluster_setting", target: 0, info: {SettingName:trace.debug.enable Value:false User:root}
I180720 05:22:08.646161 486 storage/replica_command.go:279  [split,n1,s1,r12/1:/{Table/15-Max}] initiating a split of this range at key /Table/16 [r13]
I180720 05:22:08.651789 420 storage/replica_command.go:279  [split,n1,s1,r13/1:/{Table/16-Max}] initiating a split of this range at key /Table/17 [r14]
W180720 05:22:08.655179 505 storage/intent_resolver.go:642  [n1,s1] failed to push during intent resolution: failed to push "split" id=6330a404 key=/Local/Range/Table/16/RangeDescriptor rw=true pri=0.01709992 iso=SERIALIZABLE stat=PENDING epo=0 ts=1532064128.651803845,0 orig=1532064128.651803845,0 max=1532064128.651803845,0 wto=false rop=false seq=1
I180720 05:22:08.659034 520 sql/event_log.go:126  [n1,intExec=initializeClusterSecret] Event: "set_cluster_setting", target: 0, info: {SettingName:cluster.secret Value:gen_random_uuid()::STRING User:root}
I180720 05:22:08.664582 509 storage/replica_command.go:279  [split,n1,s1,r14/1:/{Table/17-Max}] initiating a split of this range at key /Table/18 [r15]
I180720 05:22:08.667786 496 storage/replica_command.go:279  [split,n1,s1,r15/1:/{Table/18-Max}] initiating a split of this range at key /Table/19 [r16]
I180720 05:22:08.683928 562 storage/replica_command.go:279  [split,n1,s1,r16/1:/{Table/19-Max}] initiating a split of this range at key /Table/20 [r17]
I180720 05:22:08.684690 528 sql/event_log.go:126  [n1,intExec=create-default-db] Event: "create_database", target: 50, info: {DatabaseName:defaultdb Statement:CREATE DATABASE IF NOT EXISTS defaultdb User:root}
I180720 05:22:08.687898 578 storage/replica_command.go:279  [split,n1,s1,r17/1:/{Table/20-Max}] initiating a split of this range at key /Table/21 [r18]
I180720 05:22:08.688459 549 sql/event_log.go:126  [n1,intExec=create-default-db] Event: "create_database", target: 51, info: {DatabaseName:postgres Statement:CREATE DATABASE IF NOT EXISTS postgres User:root}
I180720 05:22:08.691038 69 server/server.go:1588  [n1] done ensuring all necessary migrations have run
I180720 05:22:08.691068 69 server/server.go:1591  [n1] serving sql connections
I180720 05:22:08.693999 610 server/server_update.go:65  [n1] no need to upgrade, cluster already at the newest version
I180720 05:22:08.694536 612 sql/event_log.go:126  [n1] Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:127.0.0.1:39727} Attrs: Locality: ServerVersion:2.0-9} ClusterID:79815005-6dab-4be9-bf0d-e3ba2d786de4 StartedAt:1532064128554499271 LastUp:1532064128554499271}
I180720 05:22:08.694849 545 storage/replica_command.go:279  [split,n1,s1,r18/1:/{Table/21-Max}] initiating a split of this range at key /Table/22 [r19]
I180720 05:22:08.699769 584 storage/replica_command.go:279  [split,n1,s1,r19/1:/{Table/22-Max}] initiating a split of this range at key /Table/23 [r20]
I180720 05:22:08.703331 559 storage/replica_command.go:279  [split,n1,s1,r20/1:/{Table/23-Max}] initiating a split of this range at key /Table/50 [r21]
I180720 05:22:08.706787 637 storage/replica_command.go:279  [split,n1,s1,r21/1:/{Table/50-Max}] initiating a split of this range at key /Table/51 [r22]
I180720 05:22:08.731378 643 sql/event_log.go:126  [n1,client=127.0.0.1:37892,user=root] Event: "create_table", target: 52, info: {TableName:defaultdb.public.blah Statement:CREATE TABLE blah (i INT) User:root}
I180720 05:22:08.731993 644 storage/replica_command.go:279  [split,n1,s1,r22/1:/{Table/51-Max}] initiating a split of this range at key /Table/52 [r23]
I180720 05:22:08.732518 643 sql/lease.go:313  [n1,client=127.0.0.1:37892,user=root] publish: descID=52 (blah) version=2 mtime=2018-07-20 05:22:08.731577573 +0000 UTC
I180720 05:22:08.737239 643 sql/event_log.go:126  [n1,client=127.0.0.1:37892,user=root] Event: "drop_table", target: 52, info: {TableName:defaultdb.public.blah Statement:DROP TABLE blah User:root CascadeDroppedViews:[]}
I180720 05:22:08.746518 643 sql/lease.go:313  [n1,client=127.0.0.1:37892,user=root,scExec=] publish: descID=52 (blah) version=3 mtime=2018-07-20 05:22:08.746412668 +0000 UTC
--- FAIL: TestImportData (3.34s)

```

Failed with:

--- FAIL: TestImportData/TABLE_weather_FROM_PGDUMP:_fk_unreferenced_skipped (0.07s)
[01:23:15][Step 2/2]     	import_stmt_test.go:523: IMPORT TABLE weather FROM PGDUMP ($1) WITH skip_foreign_keys
[01:23:15][Step 2/2]     	import_stmt_test.go:527: unexpected: pq: could not mark job 366981058973171713 as succeeded: creating tables: restoring table desc and namespace entries: failed to lookup parent DB 126: descriptor not found

--- a/pkg/ccl/importccl/import_stmt_test.go
+++ b/pkg/ccl/importccl/import_stmt_test.go
@@ -138,7 +138,7 @@ d
 			typ:  "CSV",
 			data: `5,STRING,7,9`,
 			query: map[string][][]string{
-				`SELECT count(*) from d.t`: {{"1"}},
+				`SELECT count(*) from t`: {{"1"}},
 			},
 		},
 		{
@@ -148,7 +148,7 @@ d
 			data: `\x0143
 0143`,
 			query: map[string][][]string{
-				`SELECT * from d.t`: {{"\x01C"}, {"0143"}},
+				`SELECT * from t`: {{"\x01C"}, {"0143"}},
 			},
 		},
 		{
@@ -204,7 +204,7 @@ d
 			typ:    "MYSQLOUTFILE",
 			data:   "@N\nN@@\nNULL",
 			query: map[string][][]string{
-				`SELECT COALESCE(s, '(null)') from d.t`: {{"(null)"}, {"N@"}, {"NULL"}},
+				`SELECT COALESCE(s, '(null)') from t`: {{"(null)"}, {"N@"}, {"NULL"}},
 			},
 		},
 		{
@@ -214,7 +214,7 @@ d
 			typ:    "MYSQLOUTFILE",
 			data:   "\\N\n\\\\N\nNULL",
 			query: map[string][][]string{
-				`SELECT COALESCE(s, '(null)') from d.t`: {{"(null)"}, {`\N`}, {"NULL"}},
+				`SELECT COALESCE(s, '(null)') from t`: {{"(null)"}, {`\N`}, {"NULL"}},
 			},
 		},
 		{
@@ -239,7 +239,7 @@ d
 			typ:    "MYSQLOUTFILE",
 			data:   "\\N\n\\\\N\nNULL",
 			query: map[string][][]string{
-				`SELECT COALESCE(s, '(null)') from d.t`: {{`\N`}, {`\\N`}, {"(null)"}},
+				`SELECT COALESCE(s, '(null)') from t`: {{`\N`}, {`\\N`}, {"(null)"}},
 			},
 		},
 		{
@@ -248,7 +248,7 @@ d
 			typ:    "MYSQLOUTFILE",
 			data:   `\x`,
 			query: map[string][][]string{
-				`SELECT * from d.t`: {{`\x`}},
+				`SELECT * from t`: {{`\x`}},
 			},
 		},
 
@@ -273,7 +273,7 @@ d
 			typ:    "PGCOPY",
 			data:   `\x43\122`,
 			query: map[string][][]string{
-				`SELECT * from d.t`: {{"CR"}},
+				`SELECT * from t`: {{"CR"}},
 			},
 		},
 		{
@@ -282,7 +282,7 @@ d
 			typ:    "PGCOPY",
 			data:   "1\tSTR\n2\t\\N\n\\N\t\\t",
 			query: map[string][][]string{
-				`SELECT * from d.t`: {{"1", "STR"}, {"2", "NULL"}, {"NULL", "\t"}},
+				`SELECT * from t`: {{"1", "STR"}, {"2", "NULL"}, {"NULL", "\t"}},
 			},
 		},
 		{
@@ -292,7 +292,7 @@ d
 			with:   `WITH delimiter = ','`,
 			data:   "1,STR\n2,\\N\n\\N,\\,",
 			query: map[string][][]string{
-				`SELECT * from d.t`: {{"1", "STR"}, {"2", "NULL"}, {"NULL", ","}},
+				`SELECT * from t`: {{"1", "STR"}, {"2", "NULL"}, {"NULL", ","}},
 			},
 		},
 		{
@@ -565,13 +565,14 @@ COPY t (a, b, c) FROM stdin;
 	sqlDB.Exec(t, `CREATE TABLE blah (i int)`)
 	sqlDB.Exec(t, `DROP TABLE blah`)
 
-	for _, tc := range tests {
+	for i, tc := range tests {
 		t.Run(fmt.Sprintf("%s: %s", tc.typ, tc.name), func(t *testing.T) {
-			sqlDB.Exec(t, `CREATE DATABASE d; USE d`)
-			defer sqlDB.Exec(t, `DROP DATABASE d`)
+			dbName := fmt.Sprintf("d%d", i)
+			sqlDB.Exec(t, fmt.Sprintf(`CREATE DATABASE %s; USE %[1]s`, dbName))
+			defer sqlDB.Exec(t, fmt.Sprintf(`DROP DATABASE %s`, dbName))
 			var q string
 			if tc.create != "" {
-				q = fmt.Sprintf(`IMPORT TABLE d.t (%s) %s DATA ($1) %s`, tc.create, tc.typ, tc.with)
+				q = fmt.Sprintf(`IMPORT TABLE t (%s) %s DATA ($1) %s`, tc.create, tc.typ, tc.with)
 			} else {
 				q = fmt.Sprintf(`IMPORT %s ($1) %s`, tc.typ, tc.with)
 			}
@@ -588,11 +589,10 @@ COPY t (a, b, c) FROM stdin;
 	}
 
 	t.Run("mysqlout multiple", func(t *testing.T) {
-		sqlDB.Exec(t, `CREATE DATABASE d; USE d`)
-		sqlDB.Exec(t, `DROP TABLE IF EXISTS d.t`)
+		sqlDB.Exec(t, `CREATE DATABASE mysqlout; USE mysqlout`)
 		dataString = "1"
-		sqlDB.Exec(t, `IMPORT TABLE d.t (s STRING) MYSQLOUTFILE DATA ($1, $1)`, srv.URL)
-		sqlDB.CheckQueryResults(t, `SELECT * FROM d.t`, [][]string{{"1"}, {"1"}})
+		sqlDB.Exec(t, `IMPORT TABLE t (s STRING) MYSQLOUTFILE DATA ($1, $1)`, srv.URL)
+		sqlDB.CheckQueryResults(t, `SELECT * FROM t`, [][]string{{"1"}, {"1"}})
 	})
 }
 

